package.json

```
@@ -7,7 +7,7 @@
     "build": "tsc",
     "lint": "eslint .",
     "start": "node dist/src/index.js",
-    "test": "jest --colors --testPathIgnorePatterns dist --testTimeout 5000"
+    "test": "jest --colors --testPathIgnorePatterns dist --testTimeout 5000 --runInBand"
```

¿Esto se te escapó? Levanta el tiempo que tarda en correr los tests. Si te jode para correrlos local, podemos tener otro script que corra secuencial:

```suggestion
    "testCI": "jest --colors --testPathIgnorePatterns dist --testTimeout 5000",
    "test": "npm run test -- --runInBand"
```

Creo que era así dejando doble guión para que le pase el parámetro "adentro" (habría que modificar el travis para q le pegue al nuevo script).

Igual lo dice alguien que corrió estos tests dos veces.

---

Lo puse secuencial porque como los tests le pegan a la misma db no se puede paralelizar :-1: